cl/citations/annotate_citations.py

```diff
@@ -15,10 +16,18 @@ def get_and_clean_opinion_text(document: Opinion | RECAPDocument) -> None:
 
     :param document: The Opinion or RECAPDocument whose text should be parsed
     """
-    for attr in ["html_anon_2020", "html_columbia", "html_lawbox", "html"]:
+    for attr in [
```

Comment:
Are these in priority order? It will pick up the first one available and skip the others. 

I think this should have a comment to make it clear

Reply:
its hard to say a correct order - honestly, it mostly matters for harvard being above columbia - otherwise the rest is pretty much the same